examples/simple.rs

```diff
@@ -1,8 +1,15 @@
-use spinoff::{Color, Spinner, Spinners};
+#![allow(unused_imports)]
+use spinoff::{spinners, Color, Spinner};
 use std::{thread::sleep, time::Duration};
 
+#[cfg(feature = "dots")]
```

Comment:
If the default feature is set to `all`, everything is working fine, and it doesn't even need this line. However, if the default is set to something like `arc`, I have no clue how to actually enable the `dots` feature.

Reply:
I'll add documentation!